README.md

```diff
@@ -1,7 +1,5 @@
 # Constant Value Inference (Interval Inference)
 
-[![build-status](https://travis-ci.org/txiang61/cast_checker.svg?branch=master)](https://travis-ci.org/txiang61/cast_checker)
```

Comment:
Wouldn't this be nice, when change to `value-inference`?

Reply:
Yes, if we decide to go with Travis, then it is nice to keep this.